Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM alpine as make_dist
+
+RUN apk update && apk add \
+    make \
+    bash \
+    tar \
+    python3
+
+# If build_src is alredy present, shouldn't need to redownload
+COPY build_src build_src
+COPY Makefile .
+
+RUN make clean && make dist/kolibri_archive.tar.gz build_src/VERSION
+
+
+FROM ubuntu:bionic
+
+# Fetch some additional build requirements
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+      adduser \
+      build-essential \
+      devscripts \
+      dirmngr \
+      fakeroot \
+      software-properties-common
+
+# Use the published kolibri-proposed PPA
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys AD405B4A && \
+    add-apt-repository -y -u -s ppa:learningequality/kolibri-proposed
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y build-dep kolibri
```

Comment:
In the unlikely event that we change the build dependencies, this method would fail.

But good news! We can also install them from the `debian/control` file:

```
# Installs the build dependencies specified in debian/control
mk-build-deps --install --tool='apt-get -o Debug::pkgProblemResolver=yes --no-install-recommends --yes' debian/control
```

Reply:
This also means that we can eliminate the need to add a PPA and run `apt-get update` :)

Reply:
It was also mentioned that `apt install equivs` might be necessary: https://stackoverflow.com/a/19432425/405682

Reply:
You mind adding this in a later PR? I'm primarily porting the build scripts over from the Kolibri repo, and would like this to be in, at least as a first pass, ASAP.